NEWS

```diff
@@ -80,15 +86,16 @@ New Functionality
 	$ zeek ./hello.js
 	Hello, Zeek!
 
-  When a recent version of the libnode package is installed, the externally
-  maintained ZeekJS plugin (https://github.com/corelight/zeekjs) is automatically
-  included as a builtin plugin. This allows Zeek to load and execute execute
-  JavaScript code located in ``.js`` or ``.cjs`` files. When no such files are
-  passed to Zeek, the JavaScript engine and Node.js environment aren't initialized
-  and there is no runtime impact.
+  When building Zeek on a system that features a recent (16.13+) version of the
+  libnode package with development headers, Zeek automatically includes the
+  externally maintained ZeekJS plugin (https://github.com/corelight/zeekjs) as a
```

Comment:
```suggestion
  externally-maintained ZeekJS plugin (https://github.com/corelight/zeekjs) as a
```

Reply:
You wouldn't believe the number of beers academics have consumed debating that one. ;-)